integration_tests/src/lib.rs

```diff
@@ -99,13 +431,17 @@ mod test {
 
             Ok(TestDownstairs {
                 address,
-                tempdir,
+                dataset,
                 downstairs: Some(downstairs),
             })
         }
 
+        pub fn path(&self) -> Result<Box<Path>> {
```

Comment:
Why return a `Box<Path>` instead of a `PathBuf` here?

Reply:
I don't know! I remember some type troubles but it compiles just fine returning PathBuf here. Changed in b6e90bce